configs/cdappconfig.json

```diff
@@ -0,0 +1,63 @@
+{
+  "database": {
+    "adminPassword": "postgres",
+    "adminUsername": "postgres",
+    "hostname": "localhost",
+    "name": "idmsvc",
+    "password": "idmsvc",
+    "port": 5432,
+    "sslMode": "disable",
+    "username": "idmsvc"
+  },
+  "endpoints": [
+    {
+      "apiPath": "/api/idmsvc/",
+      "apiPaths": [
+        "/api/idmsvc/"
+      ],
+      "app": "idmsvc-backend",
+      "hostname": "localhost",
+      "name": "service",
+      "port": 8000,
+      "tlsPort": 0
+    }
+  ],
+  "featureFlags": {
+    "hostname": "localhost",
+    "port": 4242,
+    "scheme": "http"
+  },
+  "kafka": {
+    "brokers": [
+      {
+        "hostname": "localhost",
+        "port": 9092
+      }
+    ],
+    "topics": []
+  },
```

Comment:
On stage, the clowder config does not have a Kafka entry. See #97

Reply:
Thanks, updated!